hypothesis-python/docs/data.rst

```diff
@@ -198,7 +198,7 @@ for your data type, returns a new strategy for it. So for example:
     {'de(l': None,
      'nK': {'(Rt)': None,
             '+hoZh1YU]gy8': True,
-            '8z]EIFA06^l`i^': 'LFE{Q',
+            '8z]EIFA06^li^': 'LFE{Q',
```

Comment:
This is just because the backtick broke syntax highlighting support in VS code, and I wanted to delay new contributors' first encounter with the Curse of Hypothesis.